client/my-sites/stats/stats-purchase/stats-purchase-tier-upgrade-slider.tsx

```diff
@@ -0,0 +1,73 @@
+import { PricingSlider } from '@automattic/components';
+import classNames from 'classnames';
+import { translate } from 'i18n-calypso';
+import React, { useState } from 'react';
+import useAvailableUpgradeTiers from './stats-purchase-tier-upgrade-slider-utils';
+
+import './stats-purchase-tier-upgrade-slider.scss';
+
+function getLocalizedStrings() {
+	const limits = translate( 'Monthly site views limit', {
+		comment: 'Heading for Stats Upgrade slider. The monthly views limit.',
+	} );
+	const price = translate( 'You will pay', {
+		comment: 'Heading for Stats Upgrade slider. The monthly price.',
+	} );
+	const strategy = translate( 'Price per month, billed yearly', {
+		comment: 'Stats Upgrade slider message. The billing strategy.',
+	} );
+
+	return {
+		limits,
+		price,
+		strategy,
+	};
+}
+
+type TierUpgradeSliderProps = {
+	className?: string;
+};
+
+function TierUpgradeSlider( { className }: TierUpgradeSliderProps ) {
+	const componentClassNames = classNames( 'stats-tier-upgrade-slider', className ? className : '' );
```

Comment:
I don't think we need the ternary operator. If you pass an undefined value to `classnames` I think it would skip it.